easybuild/easyconfigs/e/EvidentalGene/EvidentalGene-2018.01.01-intel-2017a-Perl-5.24.1.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = "PackedBinary"
+
+name = "EvidentalGene"
```

Comment:
@hajgato - the `name` should be `EvidentialGene`, surely?

Reply:
@verdurin For me it can be anything, its hard to figure it out what to give. The homepage references many times to `EvidentalGene` as well as `evigene`.

Reply:
@verduin I must be blind. Typo corrected.